Source/VolunteerReporting/Read/CaseReportsForListing/CaseReportForListing.cs

```diff
@@ -0,0 +1,34 @@
+using Concepts;
+using Read.CaseReports;
+using Read.DataCollectors;
+using Read.HealthRisks;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Read.CaseReportsForListing
+{
+    public class CaseReportForListing
+    {
+        public Guid Id { get; private set; }
+        public CaseReportStatus Status { get; internal set; }
+        public DataCollectorId DataCollectorId { get; internal set; }
+        public string DataCollectorDisplayName { get; internal set; } = "Unknown"; //QUESTION: Should this be a concept with default value if unknown?
+        public HealthRiskId HealthRiskId { get; internal set; }
+        public string HealthRisk { get; internal set; } = "Unknown"; //QUESTION: Should this be a concept with default value if unknown?
+        public int? NumberOfFemalesOver5 { get; internal set; } //QUESTION: Should this be a concept with default value if unknown?
+        public int? NumberOfFemalesUnder5 { get; internal set; } //QUESTION: Should this be a concept with default value if unknown?
```

Comment:
Is there a specific reason why these properties can be nullable? Isn't it more logical to have them not be nullable, but 0 instead if that value have no meaning?

Reply:
I made them nullable to make it more visible that the case report was not parsed correctly, so we do not know what the number should be. You also have something called zero reports, but since that would have status success, it would be easy to differentiate between them. I can change this and update if I get any better tips later :)